app/core/models/ocelot/stanzas/SequenceStanza.scala

```diff
@@ -52,40 +52,70 @@ object SequenceStanza {
+              case _ => true
+            }
           }
           .drop(1)  // Drop the Sequence
           .collect{case ks @ KeyedStanza(_, s: Stanza with Evaluate) => (ks.key, ks.stanza)}
           .toMap
         // push the flows and Continuation corresponding to the checked items, then
         // nextFlow and redirect to the first flow (setting list and first flow label)
-        label.fold(labels)(l => labels.updateList(s"${l}_seq", chosenOptions))
-             .pushFlows(checked.flatMap(idx => next.lift(idx).fold[List[String]](Nil)(List(_))), next.last, label, chosenOptions, continuationStanzas)
+        getLabel.fold(labels)(l => labels.updateList(s"${l}_seq", chosenOptions))
+          .pushFlows(checked.flatMap(idx => next.lift(idx).fold[List[String]](Nil)(List(_))), next.last, getLabel, chosenOptions, continuationStanzas)
         }
       }
     }
 
   def validInput(value: String): Option[String] =
-    asListOfInt(value).fold[Option[String]](None)(l => if (l.forall(options.indices.contains(_))) Some(value) else None)
+    asListOfInt(value).fold[Option[String]](None)(l => if (l.forall(getOptions.indices.contains(_))) Some(value) else None)
+}
+
+case class NonExclusiveSequence(text: Phrase,
+                    override val next: Seq[String],
+                    options: Seq[Phrase],
+                    label: Option[String],
+                    stack: Boolean) extends Sequence {
+
+  override val labelRefs: List[String] = labelReferences(text.english) ++ options.flatMap(a => labelReferences(a.english))
+  override val labels: List[Label] = label.fold[List[Label]](Nil)(l => List(ScalarLabel(l)))
+
+  override def getOptions: Seq[Phrase] = options
+  override def getLabel: Option[String] = label
+}
+
+case class ExclusiveSequence(text: Phrase,
+                             override val next: Seq[String],
+                             options: Seq[Phrase],
+                             exclusiveOptions: Seq[Phrase],
+                             label: Option[String],
+                             stack: Boolean) extends Sequence {
+
+  override val labelRefs: List[String] = labelReferences(
+    text.english) ++ options.flatMap(a => labelReferences(a.english)) ++ exclusiveOptions.flatMap(b => labelReferences(b.english))
+  override val labels: List[Label] = label.fold[List[Label]](Nil)(l => List(ScalarLabel(l)))
+
+  override def getOptions: Seq[Phrase] = options ++ exclusiveOptions
+  override def getLabel: Option[String] = label
+}
+
+object NonExclusiveSequence {
+  def apply(s: SequenceStanza, text: Phrase, options: Seq[Phrase]): NonExclusiveSequence = NonExclusiveSequence(text, s.next, options, s.label, s.stack)
 }
 
-object Sequence {
-  def apply(s: SequenceStanza, text: Phrase, options: Seq[Phrase]): Sequence = Sequence(text, s.next, options, s.label, s.stack)
+object ExclusiveSequence {
+  def apply(s: SequenceStanza, text: Phrase, options: Seq[Phrase], exclusiveOptions: Seq[Phrase]): ExclusiveSequence =
```

Comment:
Can there be more than one exclusive option?

Reply:
At this stage yes. The number of exclusive options defined is down to the guidance writer. Validation is applied in the method "pagesWithValidation", but this method is not executed until after the pages have been built.